crypto/curve25519.go

```diff
@@ -125,6 +125,14 @@ func ed25519Verify(public ed25519PublicKey, data []byte, sig ed25519Signature) b
 // produced by a holder of a cryptographic secret.
 type Signature ed25519Signature
 
+// BlankSignature is an empty signature structure, containing nothing but zeroes
+var BlankSignature = Signature{}
+
+// Blank tests to see if the given signature contains only zeros
+func (s *Signature) Blank() bool {
```

Comment:
The function name "Blank" was chosen to align with the method of the same name used for Multisig and LogicSig